app/src/main/java/com/danlls/daniel/todule_android/activities/TimeReminderFragment.java

```diff
@@ -124,7 +124,7 @@ public void afterTextChanged(Editable editable) {
     }
 
     private void ShowTos(String msg) {
-        Toast.makeText(getActivity(), "Reminder updated "+msg,
+        Toast.makeText(getActivity(), msg,
                 Toast.LENGTH_LONG).show();
     }
 
```

Comment:
`ShowTos` function seems pretty redundant to me, doesn't really do anything except wrapping the `makeText` function.